### PR TITLE
Update phoenix to 1.5

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Summary
- Updated ex_admin to use @inner_content where appropriate as it throws warnings with phoenix 1.5

[//]: # (#### Related PRs)

#### Additional deploy notes
Deploy the needed ENV vars first https://github.com/santiment/devops/pull/1725

[//]: # (#### Screenshots)

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -126,7 +128,8 @@ defmodule Sanbase.Mixfile do
         "ecto.create -r Sanbase.Repo",
         "ecto.load -r Sanbase.Repo"
       ],
-      "ecto.migrate": [
+      "ecto.m
+      grate": [
```
Comment (tspenov) [inline mix.exs:132]: `ecto.migrate`

File: lib/sanbase_web/router.ex [hunk 2]
```diff
@@ -38,6 +38,9 @@ defmodule SanbaseWeb.Router do
 
   scope "/admin2", SanbaseWeb do
     pipe_through([:browser, :basic_auth])
+    import Phoenix.LiveDashboard.Router
+
+    live_dashboard("/dashboard", metrics: TesttWeb.Telemetry)
```
Comment (tspenov) [inline lib/sanbase_web/router.ex:43]: is `TesttWeb` a typo ?

Comment (IvanIvanoff) [reply lib/sanbase_web/router.ex:43]: yes

Comment (IvanIvanoff) [reply lib/sanbase_web/router.ex]: fixed

**Pull request merged**